apollo-runtime/src/main/java/com/apollographql/apollo/ApolloCall.java

```diff
@@ -13,25 +13,83 @@
 import javax.annotation.Nonnull;
 import javax.annotation.Nullable;
 
+/**
+ * ApolloCall is an abstraction for a request that has been prepared for execution. ApolloCall
+ * represents a single request/response pair and cannot be executed twice, though it can be cancelled.
+ */
 public interface ApolloCall<T> extends Cancelable {
 
+  /**
+   * Sends the request immediately and blocks until the response can be processed or is an error.
+   *
+   * @return The successful or failed {@link Response}
+   * @throws ApolloException       if the request could not be executed due to a cancellation, a timeout or a network
```

Comment:
or parsing error, the full list of possible error types: 
`com.apollographql.apollo.exception.ApolloHttpException`
`com.apollographql.apollo.exception.ApolloNetworkException`
`com.apollographql.apollo.exception.ApolloParseException`